doc/man3/SSL_CTX_set1_curves.pod

```diff
@@ -20,6 +20,7 @@ SSL_set1_curves_list, SSL_get1_curves, SSL_get_shared_curve
 
  int SSL_get1_groups(SSL *ssl, int *groups);
  int SSL_get_shared_group(SSL *s, int n);
+ int SSL_get_negotiated_group(SSL *s);
```

Comment:
The function also needs to be in the NAME section at the top.  This is the cause of the build failures.

Reply:
Even after updating NAME section also build is failing.

Reply:
It's because it's a macro, and our script that handles this doesn't detect macros very well, so you need to add it to `util/private.num` as well, like this:

``` diff
diff --git a/util/private.num b/util/private.num
index 1e76dfb43c..4e0646d9a6 100644
--- a/util/private.num
+++ b/util/private.num
@@ -441,6 +441,7 @@ SSL_get_max_cert_list                   define
 SSL_get_max_proto_version               define
 SSL_get_min_proto_version               define
 SSL_get_mode                            define
+SSL_get_negotiated_group                define
 SSL_get_peer_signature_nid              define
 SSL_get_peer_tmp_key                    define
 SSL_get_secure_renegotiation_support    define
```

Reply:
Yes. After updating this build is passing.